mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -37,10 +57,10 @@ CLASS alv net/minecraft/world/biome/Biome
 	FIELD h WATER Lave;
 	FIELD i BIOMES Ljava/util/Set;
 	FIELD j NAMED_BIOME_IDS Leh;
-	FIELD m DECORATOR_DOUBLE_PLANT Layb;
-	FIELD n DECORATOR_TREE Lazi;
-	FIELD o DECORATOR_TREE_LARGE_OAK Laxs;
-	FIELD p DECORATOR_TREE_OAK Lazg;
```

Comment:
Not sure if there was discussion on this, but I personally prefer prefixing them with the same thing. Makes it easier to autocomplete in an IDE.